### PR TITLE
update cqm filename from NIH, add md5summed file to database load

### PR DESCRIPTION
**Pull request by stephenwaite**
still have to get file at NIH but will simplify load, yes the cqm file loaded macaloon :), loaded about 100000 records in just under 10 mins

### CODE REVIEW

**Comment by stephenwaite** [comment]
figured there's licensing or would have placed file in contrib

File: sql/5_0_1-to-5_0_2_upgrade.sql [hunk 1]
```diff
@@ -810,6 +810,6 @@ ALTER TABLE `ar_session`
     MODIFY `patient_id` bigint(20) NOT NULL;
 #EndIf
 
-#IfMissingColumn documents encrypted
-ALTER TABLE `documents` ADD `encrypted` TINYINT(4) NOT NULL DEFAULT '0' COMMENT '0->No,1->Yes'
-#EndIf
```
Comment (bradymiller) [inline sql/5_0_1-to-5_0_2_upgrade.sql]: need to add above 3 lines back :)

File: sql/database.sql [hunk 2]
```diff
@@ -1222,7 +1222,6 @@ CREATE TABLE `documents` (
   `audit_master_approval_status` TINYINT NOT NULL DEFAULT 1 COMMENT 'approval_status from audit_master table',
   `audit_master_id` int(11) default NULL,
   `documentationOf` varchar(255) DEFAULT NULL,
-  `encrypted` TINYINT(4) NOT NULL DEFAULT '0' COMMENT '0->No,1->Yes',
```
Comment (bradymiller) [inline sql/database.sql]: need to add above line back :)

**Comment by stephenwaite** [comment]
whoops, will redo

**Pull request closed without merging**